src/markdown-pages/install-with-kurl/migrating.md

```diff
@@ -0,0 +1,89 @@
+---
+path: "/docs/install-with-kurl/migrating"
+date: "2020-07-24"
+weight: 20
+linktitle: "Migrating"
+title: "Migrating"
```

Comment:
Let's make the title more specific—maybe something like "Migrating to Change kURL Add-Ons" (just spitballing) since we're talking about migrating for a specific purpose. And then if the in-cluster migrations are doc'ed, we can add them here or make another page.

Reply:
Updated